src-docs/src/views/guidelines/getting_started.md

```diff
@@ -19,7 +19,7 @@ yarn add @opensearch-project/oui @elastic/datemath moment prop-types
 
 We depend upon the version of node defined in [.nvmrc](.nvmrc).
 
-You will probably want to install a node version manager. [nvm](https://github.com/creationix/nvm) is recommended.
+We recommend installing a node version manager: [nvm](https://github.com/creationix/nvm) is recommended.
```

Comment:
Nitpick: This link redirects to https://github.com/nvm-sh/nvm, so I think we could just have that directly.